SU-HSDO/suhumsci
# shs-132: adjust megamenu mobile to indicate active child items

## Summary
This adjusts the megamenu to let users know they are on a page they clicked on.

## Steps to Test
1. On a site with the new megamenu, go to a nested child page. I liked this one `/events/colloquium-series`.
2. In mobile view, open the menu and see that menu item it's located under is expanded and the level 2 item is highlighted (screenshot below)
3. Refresh the page in desktop view. Click on the parent page item and see that the desktop styles remain unaffected.

## Screenshots
![Screen Shot 2022-12-20 at 2 51 50 PM](https://user-images.githubusercontent.com/8405274/208755290-bc55b4d8-3948-4fb8-b8c0-673039985c45.png)

![Screen Shot 2022-12-20 at 2 51 43 PM](https://user-images.githubusercontent.com/8405274/208755309-dc4150a5-6934-4c2d-881c-651e07e105fd.png)

## Notes
The ticket says "Please update the mobile behavior to reflect the mobile menu behavior when the ‘standard’ menu is enabled." which I have, but the ticket also states "does not visually indicate which page you’re on" which the standard menu doesn't either if you're beyond level two. If that's confusing, just ping me, but wanted to call that out.
